docs(Kommander): rename cloud providers to infrastructure providers (#2991)
mesosphere/dcos-docs-site: pages/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/index.md

---
layout: layout.pug
navigationTitle: Infrastructure Providers
title: Infrastructure Providers
menuWeight: 1
excerpt: Managing infrastructure providers used by Kommander
---

> Cloud providers like AWS, Azure and Google provide the infrastructure for your Konvoy clusters. To automate their provisioning, Kommander needs authentication keys to your preferred cloud provider. You may have many accounts for a single cloud provider.

Reviewer: ```suggestion
Cloud providers like AWS, Azure, and Google provide the infrastructure for your Konvoy clusters. To automate their provisioning, Kommander needs authentication keys for your preferred cloud provider. You may have many accounts for a single cloud provider.
```

> In order to provision new clusters and manage them, Kommander needs cloud provider credentials. Currently AWS, Azure, and On Premise are supported.

Reviewer: ```suggestion
To provision new clusters and manage them, Kommander needs cloud provider credentials. Currently AWS, Azure, and On Premise are supported.
```

> In order to provision new clusters and manage them, Kommander needs cloud provider credentials. Currently AWS, Azure, and On Premise are supported.

Reviewer 2: Cloud or infrastructure here?

### Viewing and Modifying Infrastructure Providers

> Cloud Provider credentials are configured within each workspace, so you must first select a workspace. Then, navigate to the Cloud Providers option under Administration.

Reviewer: ```suggestion
Cloud Provider credentials are configured in each workspace, so you must first select a workspace. Then, navigate to the Cloud Providers option under Administration.
```

> Cloud Provider credentials are configured within each workspace, so you must first select a workspace. Then, navigate to the Cloud Providers option under Administration.

Reviewer 2: I think it should be "infrastructure" instead of "cloud" in both places.

![Empty Infrastructure Providers Page](/ksphere/kommander/1.1.0-beta/img/empty-infrastructure-providers.png)
<br />_Empty Infrastructure Providers Page_

![Infrastructure Provider Form](/ksphere/kommander/1.1.0-beta/img/add-infrastructure-provider.png)
<br />_Infrastructure Provider Form_

#### Configuring an AWS Infrastructure Provider

- [Configure AWS Provider with Role Credentials](/ksphere/kommander/latest/operations/infrastructure-providers/configure-aws-cloud-provider-roles) (Recommended)
- [Configure AWS Provider with Static Credentials](/ksphere/kommander/latest/operations/infrastructure-providers/configure-aws-cloud-provider-static-credentials)

#### Configuring an Azure Infrastructure Provider

- [Configure Azure Provider with Static Credentials](/ksphere/kommander/latest/operations/infrastructure-providers/configure-azure-cloud-provider)

#### Configuring an On-Premise Infrastructure Provider

- [Configure On-Premise Provider](/ksphere/kommander/latest/operations/infrastructure-providers/configure-on-prem-provider)

### Deleting an infrastructure provider

> Before attempting to delete an infrastructure provider, Kommander first verifies if any existing managed clusters were created using this provider. The infrastructure provider cannot be deleted until all clusters, created with the infrastructure provider, have been deleted. This ensures Kommander has access to your cloud provider to remove all resources created for a managed cluster.

Reviewer: ```suggestion
Before deleting an infrastructure provider, Kommander verifies if any existing managed clusters were created using this provider. The infrastructure provider cannot be deleted until all clusters, created with the infrastructure provider, have been deleted. This ensures Kommander has access to your cloud provider to remove all resources created for a managed cluster.
```
